src/lexer.coffee

```diff
@@ -170,6 +170,8 @@ exports.Lexer = class Lexer
        isForFrom(prev)
       tag = 'FORFROM'
       @seenFor = no
+    else if tag is 'PROPERTY' and prev and prev.spaced and prev[0] in CALLABLE and /^[g|s]et$/.test(prev[1])
```

Comment:
For the record, that's not just shorter but also more correct. `[g|s]et` would also match `|et`.